src/clj/game/cards/hardware.clj

```diff
@@ -1238,6 +1238,46 @@
                                                (same-card? card (:host target))))
                                 :type :recurring}}})
 
+(defcard "Pantograph"
+  (let [install-ability
+        {:optional
+         {:prompt "Install card with Pantograph ability?"
+          :yes-ability
+          {:async true
+           :msg "wants to install card with Pantgraph"
```

Comment:
Change this to be `:waiting-prompt`, and then remove the nesting of the `:choices` ability.